thirdeye/thirdeye-frontend/app/pods/components/rootcause-heatmap/component.js

```diff
@@ -434,13 +434,27 @@ export default Component.extend({
   },
 
   actions: {
-    onHeatmapClick(role, dimName, dimValue) {
+    onInclude(role, dimName, dimValue) {
       const { selectedUrn, onSelection } =
         this.getProperties('selectedUrn', 'onSelection');
```

Comment:
NIP: Let's be consistent and use this here as well.

import { getProperties } from '@ember/object';